iterator-test-harness/pom.xml

```diff
@@ -65,4 +65,15 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <configuration>
+          <omitVisitors>SharedVariableAtomicityDetector,ConstructorThrow</omitVisitors>
+        </configuration>
+      </plugin>
```

Comment:
Could these changes be made in the accumulo parent pom instead of in each pom?

Reply:
They could but the point of doing it in each pom is to allow us to make these changes incrementally so we don't have to fix the whole codebase at once and instead can go module by module.

Reply:
Made this change in 941c2a3 @ctubbsii. Definitely a better approach than what I had before.